code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/PopkovKirillTest.java

```diff
@@ -0,0 +1,65 @@
+package ru.unn.agile.ComplexNumber.Model;
+
+import org.junit.Test;
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+
+import static org.junit.Assert.*;
+
+public class PopkovKirillTest {
+    private final double delta = 0.001;
+
+    @Test
+    public void initComplexNumberWithEmptyConstructor() {
+        ComplexNumber cn = new ComplexNumber();
```

Comment:
'cn' is too short for name of object, it doesn't give any information of type of object, it can be anything.
Will be better to name it 'number' in all tests.